utils/draw_plot.py

```diff
@@ -66,3 +72,45 @@ def plot_acc_loss(history, output_path=None):
         plt.savefig(output_path_name)
     else:
         plt.show()
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--cm', action='store_true')
+
+    args = parser.parse_args()
+
+    if args.cm:
+
+        #feature-based svm best approach
+        cm =   [[41,  7,  8, 15],
+                [10, 44,13,  4],
+                [ 5,  9, 35, 18],
+                [12,  6, 21, 34]]
+        
+        output_path = os.path.join('models', 'lyric', 'feature_based', 'svm')
+        
+        #feature-based ann best approach
```

Comment:
I don't like this manual filling in the CMs. It should've been invoked, for example at the end of each training.
However, if you need it may stay for now, however i would avoid doing sth liike that

Reply:
I need it, because I'm using it to plot from cluster logs